src/main/java/cz/muni/fi/pv242/rest/model/ReservationCreateDTO.java

```diff
@@ -0,0 +1,24 @@
+package cz.muni.fi.pv242.rest.model;
```

Comment:
ten ReservationCreateDTO je jedno jestli se přidá nebo ne, je to stejné jako v mastru